nostotagging.php

```diff
@@ -188,64 +204,83 @@ public function uninstall()
 	 */
 	public function getContent()
 	{
+		// We don't want the browser to cache the config page, as the iframe authentication won't work if cached.
+		header('Cache-Control: no-store, no-cache, must-revalidate, max-age=0');
+		header('Cache-Control: post-check=0, pre-check=0', false);
+		header('Pragma: no-cache');
```

Comment:
Okey, had a deeper look at this and there is actually 2 parts to this problem. The first is that the parent page would have to include those headers, as we do a server-to-server request on the page load that authenticates the user and fetches a new url for the iframe with a new token in it. But, regardless of that the iframe src changes on page load, the browser still caches the iframe content even if that page also includes the no cache headers. So the only way around that is to do it in JS, as you pointed out. But that does not remove the need for the no cache headers on the parent page. However, we will be removing the headers anyways, as it is far simpler and the only difference from the current functionality is that the parent page is not cached and we do not make server-to-server calls when navigating back/forth in the browser. The content of the iframe won't be affected as it was cached anyways from the beginning.

Should I add the change to this pull-request by just doing a new push on master?